std/experimental/allocator/package.d

```diff
@@ -1526,6 +1526,23 @@ if (is(T == class) || is(T == interface))
             static assert(!is(T: IUnknown), "COM interfaces can't be destroyed in "
                 ~ __PRETTY_FUNCTION__);
         }
+        static if (__traits(allMembers, T).length)
```

Comment:
I don't see why C++ interfaces cannot be deallocated by an allocator. The fact that you can't delete C++ interface just means that one cannot allocate/delete it using built-in new/delete (apparently).